tests/test_pynapple.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import sys
+
+
+def test_import_pynapple_and_set_backend():
+    import pynapple as nap
+    nap.nap_config.set_backend("jax")
```

Comment:
this might be 

```suggestion
    nap.nap_config.set_backend("numba")
    assert nap.nap_config.backend == "numba"
    nap.nap_config.set_backend("jax")
    assert nap.nap_config.backend == "numpy"
```